src/main/java/com/cds/ohouse/service/CheckListServiceImpl.java

```diff
@@ -32,4 +32,11 @@ public CheckListUpdateResponseDTO updateCheckList(CheckListUpdateRequestDTO chec
 
         return CheckListUpdateResponseDTO.of(checkListUpdateRequestDTO, CheckListUpdateResponseVO.of(tag));
     }
+    @Override
+    @Transactional
+    public void deleteCheckList(Long id){
+        CheckList checkList = checkListRepository.findById(id)
```

Comment:
```suggestion
        val checkList = checkListRepository.findById(id)
```
lombok.val을 써보는것은 어떨까요? val은 final이라 불변성을 보장합니다